sdk/eventhub/eventhubs-checkpointstore-tables/README.md

```diff
@@ -0,0 +1,213 @@
+# Azure Event Hubs Checkpoint Store client library for JavaScript
+
+An Azure Table storage based solution to store checkpoints and to aid in load balancing when using `EventHubConsumerClient` from the [@azure/event-hubs](https://www.npmjs.com/package/@azure/event-hubs) library
+
+[Source code](https://github.com/Azure/azure-sdk-for-js/tree/master/sdk/eventhub/eventhubs-checkpointstore-tables) | [Package (npm)](https://www.npmjs.com/package/@azure/eventhubs-checkpointstore-tables) | [API Reference Documentation](https://docs.microsoft.com/javascript/api/@azure/eventhubs-checkpointstore-tables/) | [Samples](https://github.com/Azure/azure-sdk-for-js/tree/master/sdk/eventhub/eventhubs-checkpointstore-tables/samples)
+
+## Getting started
+
+### Install the package
+
+Install the Azure Event Hubs Checkpoint Store Tables library using npm
+
+`npm install @azure/eventhubs-checkpointstore-tables`
+
+**Prerequisites**: You must have an [Azure subscription](https://azure.microsoft.com/free/), an
+[Event Hubs Namespace](https://docs.microsoft.com/azure/event-hubs/) to use this package, and a [Storage account](https://docs.microsoft.com/en-us/azure/storage/tables/table-storage-overview)
```

Comment:
We should omit the language identifier from docs.microsoft.com links so that when a user navigates to the docs, the proper language is chosen based on their own locale/settings.
`https://docs.microsoft.com/en-us/azure/storage/tables/table-storage-overview` -> `https://docs.microsoft.com/azure/storage/tables/table-storage-overview`

Update Prerequisites to use a list instead to match the `@azure/eventhubs-checkpointstore-blob` readme.

Reply:
done